src/lib.rs

```diff
@@ -7,7 +7,7 @@ mod format;
 mod levels;
 mod memtable;
 mod ops;
-mod opt;
+pub mod opt;
```

Comment:
How about also reexporting it? So users don't need to write two lines for importing from agatedb.